### ISSUE TITLE
Short title retrieved by API mismatches with the one given on website

### ISSUE THREAD

**Issue by GaurangTandon** [CrossRef/rest-api-doc#360]
**Paper:** https://pubs.acs.org/doi/abs/10.1021/ci00024a006?journalCode=jcics1
**Data received on API call** (at `https://api.crossref.org/works/10.1021/ci00024a006`):
```
DOI :"10.1021/ci00024a006",
ISSN: ["1549-9596"],
URL: "http://dx.doi.org/10.1021/ci00024a006",
alternative-id: ["ci00024a006"],
author: (4) [{…}, {…}, {…}, {…}],
container-title: ["Journal of Chemical Information and Modeling"],
content-domain: {domain: Array(0), crossmark-restriction: false},
created: {date-parts: Array(1), date-time: "2005-03-17T12:41:19Z", timestamp: 1111063279000},
deposited: {date-parts: Array(1), date-time: "2016-09-04T17:53:35Z", timestamp: 1473011615000},
indexed: {date-parts: Array(1), date-time: "2018-02-05T10:51:18Z", timestamp: 1517827878967},
is-referenced-by-count: 77,
issn-type: [{…}]
issue: "2"
issued: {date-parts: Array(1)}
link: [{…}]
member: "316"
original-title: []
page: "203-210"
prefix: "10.1021"
published-print: {date-parts: Array(1)}
publisher: "American Chemical Society (ACS)"
reference-count: 0
references-count: 0,
relation: {},
score: 1,
short-container-title: "J. Chem. Inf. Model.",
short-title: Array(0),
source: "Crossref",
subject: (4) ["General Chemistry", "General Chemical Engineering", "Library and Information Sciences", "Computer Science Applications"],
subtitle: [],
title: ["Aromatic Character of the Benzene Ring Present in …carbons. Nonequivalence of Indices of Aromaticity"],
type: "journal-article",
volume: "35"
```

**Problem:** property `short-container-title` gives the value `J. Chem. Inf. Model.`. However, a simple glance at the [official webpage](https://pubs.acs.org/doi/abs/10.1021/ci00024a006?journalCode=jcics1) gives the value as `J. Chem. Inf. Comput. Sci.,` instead.

This issue doesn't occur with a [latest 2018 paper](https://pubs.acs.org/doi/10.1021/acs.jcim.7b00410), as well as an [old 1996 paper](https://pubs.acs.org/doi/10.1021/ci960366o) (not belonging to legacy archives), so I guess this maybe an issue with ACS Legacy Archives papers only.

**Comment by GaurangTandon** [comment]
**Possibly a related issue, possibly not:**

**Retrieving the DOI: ** `10.1002/anie.196907811` via `/works/` gives the `.message.title` field as `"The Conservation of Orbital Symmetrya"`.  
Not only does the last word make no sense, the typo also is not present on the [official website](https://onlinelibrary.wiley.com/doi/abs/10.1002/anie.196907811) for this DOI.

I have no clue how that `a` crept in it. Thanks for the great work maintaining this API, hope this would be an easy fix!

**Comment by pdavis8** [comment]
Hello,
Crossref metadata is sent to us by the publisher, here they have sent in the incorrect title for the article. I will contact the publisher to ask them to redeposit the correct title for this article.
Thanks for bringing this to our attention.

**Comment by pdavis8** [comment]
Hello,
I have just been contacted by the publisher  to say that the typo has been corrected for ```10.1002/anie.196907811``` in our system now.

**Comment by GaurangTandon** [comment]
Thanks @pdavis8! Though please also see the other DOI `10.1021/ci00024a006` that I mentioned in my original report.

Also, if I come across such minor typos in the future, is there another way to get them fixed instead of having to create a new GitHub issue everytime?
